MediaGallery/MediaFile/MediaFile.android.cs

```diff
@@ -14,6 +14,7 @@ internal MediaFile(string fileName, Uri uri)
             NameWithoutExtension = Path.GetFileNameWithoutExtension(fileName);
             Extension = Path.GetExtension(fileName);
             ContentType = MimeTypeMap.Singleton.GetMimeTypeFromExtension(Extension);
+            MediaIdentifier = uri.ToString();
```

Comment:
How can we use this? Have you tried opening a file using this URI?

Reply:
No I have not. The uri.ToString() was already used. I need to spend some time to check what is needed for Android.